test/data/allele_association_ingest_test.json

```diff
@@ -28,8 +28,8 @@
       "internal": false,
       "obsolete": false,
       "taxon_curie": "NCBITaxon:6239",
-      "date_created": "2015-04-09T10:15:30+00:00",
-      "date_updated": "2022-07-11T13:12:51+00:00",
+      "date_created": "2015-04-09",
```

Comment:
@sierra-moxon Is there a reason for these date changes?  Our Java code expects an OffsetDateTime for all date fields

Reply:
ok awesome - we can change to datetime in linkml (right now these are all "date"s)